src/components/Navbar.jsx

```diff
@@ -1,73 +1,77 @@
-import React, { useState } from "react";
-import { useLocation, Link, useHistory } from "react-router-dom";
-import BoardIcon from "../assets/icons/BoardIcon";
-import DashboardIcon from "../assets/icons/DashboardIcon";
-import HomeIcon from "../assets/icons/HomeIcon";
-import SettingsIcon from "../assets/icons/SettingsIcon";
-import LogoutIcon from "../assets/icons/LogoutIcon";
-import DefaultAvatar from "../assets/images/default-avatar.png";
-import { signOut } from "../services/authService";
+import React, { useState } from 'react';
+import { useLocation, Link, useHistory } from 'react-router-dom';
+import BoardIcon from '../assets/icons/BoardIcon';
+import DashboardIcon from '../assets/icons/DashboardIcon';
+import HomeIcon from '../assets/icons/HomeIcon';
+import SettingsIcon from '../assets/icons/SettingsIcon';
+import LogoutIcon from '../assets/icons/LogoutIcon';
+import DefaultAvatar from '../assets/images/default-avatar.png';
+import { signOut } from '../services/authService';
 
 function Navbar() {
-  const history = useHistory();
-  const [hovering, setHovering] = useState(false);
-  const location = useLocation();
+	const history = useHistory();
+	const [hovering, setHovering] = useState(false);
+	const location = useLocation();
 
-  const signOutHandler = () => {
-    signOut();
-    history.push("/");
-  };
+	const signOutHandler = () => {
+		signOut();
+		history.push('/');
+	};
 
-  return (
-    <div className="navbar">
-      <div>
-        <Link to="/home" className="button-link">
-          <div className="navbar__icon__group" data-testid="nav-home-icon">
-            {location.pathname === "/home" && <div className="selected-box" />}
-            <HomeIcon />
-          </div>
-        </Link>
-        <Link to="/dashboard" className="button-link">
-          <div className="navbar__icon__group" data-testid="nav-dashboard-icon">
-            {location.pathname === "/dashboard" && (
-              <div className="selected-box" />
-            )}
-            <DashboardIcon />
-          </div>
-        </Link>
-        <Link to="/board" className="button-link">
-          <div
-            className="navbar__icon__group"
-            data-testid="nav-trelloboard-icon"
```

Comment:
Would you be able to add these data-testid tags back in? We need them for testing the navigation (not that that's working right now lol)